plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/TestElasticsearchBackpressure.java

```diff
@@ -50,7 +50,9 @@ protected QueryRunner createQueryRunner()
                 ImmutableMap.of(),
                 // This test can only run on a single node, otherwise each node exports its own stats beans and they override each other
                 // You can only bind one such bean per JVM, so this causes problems with statistics being 0 despite backpressure handling
-                1);
+                1,
+                // Use a unique catalog name to make sure JMX stats beans are unique and not affected by other tests
+                "elasticsearch-backpressure");
```

Comment:
Should we use something like `io.trino.testing.sql.TestTable#randomTableSuffix`? Or is it overkill?

Reply:
We might, I decided to go with a static name since this is executed just once for just one test.

A different approach could be to make the catalog name randomized on the `ElasticsearchQueryRunner` level, but since it was just the backpressure test causing the issue I decided to limit the changes to that test.